simplivity/resources/backups.py

```diff
@@ -213,5 +213,17 @@ def rename(self, new_name, timeout=-1):
         data = {'backup_name': new_name}
         self._client.do_post(resource_uri, data, timeout)
         self.__refresh()
+        return self
+
+    def cancel(self, timeout=-1):
+        """Cancels the specified running backup
+        Args:
+          timeout: Time out for the request in seconds.
 
+        Returns:
+          object: Backup object.
+        """
+        resource_uri = "{}/{}/cancel".format(URL, self.data["id"])
+        self._client.do_post(resource_uri, None, timeout)
+        self.__refresh()
```

Comment:
Will refresh work if the cancel operation is successful?

Reply:
Yes refresh will work, if the cancel operation on backup whose creation is in-progress.
Object is created in the backend and the status for this object is "cancelled". 
If the cancel operation is not successful the POST command will throw the exception.